.github/workflows/deploy_branch_preview.yml

```diff
@@ -149,17 +164,29 @@ jobs:
           APP_BUILD_VERSION: ${{ github.run_number }}
           APP_BINARY_VERSION: ${{ steps.latestBinaryVersion.outputs.version }}
           DOMAIN_FORMAT: ${{ secrets.TEST_DOMAIN_FORMAT }}
-        run: expo publish --non-interactive --release-channel storybook-pr${{ github.event.number }}
+          PROJECT_ID: ${{ secrets.EXPO_PROJECT_ID }}
+          START_STORYBOOK: true
+        run: eas update --non-interactive --branch storybook-pr${{ github.event.number }} --message "${{ steps.getCommitMessage.outputs.commit_message }}" --json | jq '.[].id' -r > updates.txt
+          ANDROID_URL="$(sed -n 1p updates.txt)"
+          IOS_URL="$(sed -n 2p updates.txt)"
+          echo "::set-output name=androidUrl::$ANDROID_URL"
+          echo "::set-output name=iosUrl::$IOS_URL"
       - name: Add Comment To PR
         uses: mshick/add-pr-comment@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
-          EXPO_PROJECT: "@supplyallytest/rationally"
         with:
           message: |
-            ## Application
-            ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/${{ env.EXPO_PROJECT }}?release-channel=pr${{ github.event.number }})
-            Published to https://exp.host/${{ env.EXPO_PROJECT }}?release-channel=pr${{ github.event.number }}
-            ## Storybook
-            ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/${{ env.EXPO_PROJECT }}?release-channel=storybook-pr${{ github.event.number }})
-            Published to https://exp.host/${{ env.EXPO_PROJECT }}?release-channel=storybook-pr${{ github.event.number }}
+            ## Andriod Application
```

Comment:
https://github.com/expo/expo-github-action/issues/189 eas update doesnt generate a single qr that's useable by expo go anymore. So have to manually get the qr of the update of the 2 os instead